tests/App/templates/base.html.twig

```diff
@@ -17,7 +17,7 @@
         {% endblock %}
     {% endblock %}
 </head>
-<body class="{% block body_class %}{% endblock %}" data-turbo="false">
+<body class="{% block body_class %}{% endblock %}" data-turbo="{{ araise_turbo_enabled ? 'true' : 'false' }}">
```

Comment:
create another twig extension for the parameter `araise_crud.enable_turbo` and use this one.